bom/runtime/pom.xml

```diff
@@ -1159,6 +1166,11 @@
                 <artifactId>microprofile-config-api</artifactId>
                 <version>${microprofile-config-api.version}</version>
             </dependency>
+            <dependency>
```

Comment:
Any reason the API needs to be a direct dependency?

Reply:
This is doing dependency management. The actual dependency is already set elsewhere in Quarkus, for example in the quarkus-arc extension. The problem is that previously it was relying on resteasy to control that dependency version.

Reply:
It was relying on RESTEasy? I find that odd considering it would be brought in through SmallRye Context Propagation

Reply:
The odds of Maven, I'd say. Maybe the dependency path through RESTEasy was simply shorter and hence considered first by Maven.